### ISSUE TITLE
v0.28.2 release bundle_crd.zip CRDs errror: `The CustomResourceDefinition "vmalertmanagerconfigs.operator.victoriametrics.com" is invalid`

### ISSUE THREAD

**Issue by qikiqi** [VictoriaMetrics/operator#521]
Downloaded the v0.28.2 release bundle_crd.zip, tried to apply (upgrading from v0.28.1) but receiving the following error on apply:

```

The CustomResourceDefinition "vmalertmanagerconfigs.operator.victoriametrics.com" is invalid: spec.validation.openAPIV3Schema.properties[spec].properties[route].properties[-].items.type: Required value: must not be empty for specified array items
```

So I guess the issue is with this bit that has changed since v0.28.1 in the `crd.yaml`

```
+                  '-':
+                    description: Child routes.
+                    items: {}
+                    type: array
```

Additionally, no issues if I use the [CRD from repo with tag v0.28.2](https://github.com/VictoriaMetrics/operator/blob/b6f7f56be24ef62c02b3686aed04c2a8a388e504/config/crd/bases/operator.victoriametrics.com_vmalertmanagerconfigs.yaml) (but also no changes when upgrading from v0.28.1, so not entirely sure what has been the intention)

**Comment by lujiajing1126** [comment]
Same issue

**Comment by f41gh7** [comment]
Yes, it's an issue with generated yaml. Will fix it soon. CRD from 0.28.1 release must work with 0.28.2.

 I expect, that `[]*Route` will be ignored, but it's not. Structured `[]*Route` was replaced with `[] rawJson`. It's needed for additional context during `CRD` parsing. Main issue with `[]*Route` - it contains recursive loop reference.

**Comment by f41gh7** [comment]
Must be fixed at v0.28.3 version

**Comment by qikiqi** [comment]
> Must be fixed at v0.28.3 version

Thanks, v0.28.3 fixes indeed.
